### PR TITLE
[docs] python snippets for devices

### PR DESCRIPTION
**Pull request by AlexeyLebedev1**
### Tickets:
 - *82287*

### CODE REVIEW

File: docs/OV_Runtime_UG/supported_plugins/CPU.md [hunk 1]
```diff
@@ -113,6 +183,22 @@ Therefore, if the input data shape is not changed between inference calls, it is
...
+       :language: cpp
+       :fragment: [static_shape]
+
+.. tab:: Python
+
+    .. doxygensnippet:: docs/snippets/cpu/dynamic_shape.py
+       :language: python
+       :fragment: [static_shape]
+
+@endsphinxdirective
```
[the remark this answers is not in the document]
Comment (ntyukaev) [reply docs/OV_Runtime_UG/supported_plugins/CPU.md:200]: Please use a different approach for creating sphinx tabs with code example. https://github.com/openvinotoolkit/openvino/blob/releases/2022/1/docs/OV_Runtime_UG/ov_infer_request.md#creating-infer-request

[the remark this answers is not in the document]
Comment (AlexeyLebedev1) [reply docs/OV_Runtime_UG/supported_plugins/CPU.md]: updated.

File: docs/OV_Runtime_UG/supported_plugins/GPU.md [hunk 2]
```diff
@@ -141,7 +181,17 @@ For example, batch size 33 may be executed via 2 internal networks with batch si
...
+@snippet docs/snippets/gpu/dynamic_batch.cpp dynamic_batch
+@endsphinxtab
+
+@sphinxtab{Python}
+@snippet docs/snippets/gpu/dynamic_batch.py dynamic_batch
+@endsphinxtab
+
+@endsphinxtabset
 
 See [dynamic shapes guide](../ov_dynamic_shapes.md) for more details.
```
[the remark this answers is not in the document]
Comment (ntyukaev) [reply docs/OV_Runtime_UG/supported_plugins/GPU.md:196]: you have to use "@sphinxtabset" inside "@sphinxtab"

[the remark this answers is not in the document]
Comment (AlexeyLebedev1) [reply docs/OV_Runtime_UG/supported_plugins/GPU.md:216]: preprocessing snippet is added.

File: docs/OV_Runtime_UG/supported_plugins/GPU.md [hunk 3]
```diff
@@ -48,19 +48,37 @@ Then device name can be passed to `ov::Core::compile_model()` method:
...
 
 @sphinxtab{Running on specific GPU}
 
+@sphinxtab{C++}
 @snippet docs/snippets/gpu/compile_model.cpp compile_model_gpu_with_id
+@endsphinxtab
+
+@sphinxtab{Python}
+@snippet docs/snippets/gpu/compile_model.py compile_model_gpu_with_id
+@endsphinxtab
```
[the remark this answers is not in the document]
Comment (AlexeyLebedev1) [reply docs/OV_Runtime_UG/supported_plugins/GPU.md:75]: fixed

**Review by ilya-lavrenov — APPROVED** [review]

**Review by jiwaszki — COMMENTED** [review]
LGTM

**Pull request merged**